keps/sig-cloud-provider/991-cloud-controller-migration/README.md

```diff
@@ -168,11 +174,27 @@ type ControllerLeaderConfiguration struct {
 	Component string `json:"component"`
 }
 ```
+#### Default LeaderMigrationConfiguration
+
+Both `kube-controller-manager` and `cloud-controller-manager` must each provide a default `LeaderMigrationConfiguration` 
```

Comment:
Do they really provide it or is it provided for them by the package? How is the 'X' for KCM in version X and CCM in version X+1 determined? Is it purely by then the "--enable-leader-migration" version is turned on? If so we should specify it here. Also it suggest more strongly that the package would provider the default.

Reply:
- Let's define that the `controller-manager` package provides the configuration.
- The default value is embedded in the binary, which means it's determined by the `controller-manager` version that CCM and KCM . e.g. KCM of Kubernetes `1.22.0` imports `staging/controller-manager` version `1.22.0`, and thus `staging/controller-manager` 1.22 should contain defaults for KCM of Kubernetes `1.22.0`